packages/kcms/src/main.ts

```diff
@@ -42,15 +43,26 @@ const getEnv = <C extends Context = Parameters<typeof env>['0']>(c: C) => {
   return envRes.data;
 };
 
-const jwtSecret = await crypto.subtle.generateKey(
-  {
-    name: 'ECDSA',
-    namedCurve: 'P-256',
-  },
+const { KCMS_COOKIE_SECRET, KCMS_AUTH_PRIVATE_JWK, KCMS_AUTH_PUBLIC_JWK } = process.env;
+if (!KCMS_AUTH_PRIVATE_JWK || !KCMS_AUTH_PUBLIC_JWK || !KCMS_COOKIE_SECRET) {
+  throw new Error('KCMS_AUTH_(PRIVATE/PUBLIC)_JWK/KCMS_COOKIE_SECRET is required.');
+}
+
+const authPrivateJwk = await crypto.subtle.importKey(
+  'jwk',
+  JSON.parse(KCMS_AUTH_PRIVATE_JWK),
+  { name: 'ECDSA', namedCurve: 'P-256' },
+  true,
+  ['sign']
+);
+const authPublicJwk = await crypto.subtle.importKey(
+  'jwk',
+  JSON.parse(KCMS_AUTH_PUBLIC_JWK),
+  { name: 'ECDSA', namedCurve: 'P-256' },
   true,
-  ['sign', 'verify']
+  ['verify']
 );
-const cookieSecret = crypto.getRandomValues(new Uint32Array(8));
+const cookieSecret = Buffer.from(KCMS_COOKIE_SECRET, 'base64');
```

Comment:
元が `Uint32Array` なので、一応
```suggestion
const cookieSecret = new Uint32Array(Buffer.from(KCMS_COOKIE_SECRET, 'base64'));
```
がよさそう